ReactSkia/components/RSkComponent.h

```diff
@@ -39,7 +48,7 @@ struct Component {
   EventEmitter::Shared eventEmitter{};
   LayoutMetrics layoutMetrics{EmptyLayoutMetrics};
   State::Shared state{};
-  int zIndex{};
+  struct props props_;
```

Comment:
can we follow the same way other struct members are defined.. props_ can be props and structure naming in Pascal case